ui/src/translations/en.json

```diff
@@ -230,6 +230,7 @@
     "automatic refresh": "Automatic refresh",
     "toggle periodic refresh each 10 seconds": "Toggle periodic refresh every 10 seconds",
     "trigger refresh": "Trigger refresh",
+    "add-trigger-in-editor": "Add a Trigger to your Flow first",
```

Comment:
Rename key to `no_flow_triggers` and make value sentence case.

Reply:
Could this wording be confused with Flow Triggers?

Reply:
Aren't these flow triggers? Or am I mixing something (which can very easy be the case)?

Bare in mind that this is only internal key to reference ćhe translation string value.

Reply:
This is for any triggers. But true, it's not something the user will see!

Reply:
Fair point. I'm absolutely okay with leaveing it as is then.